### PR TITLE
Request to merge form

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
###  Project Requirements

- Create forms to perform the following functions:
- Creates a Post on behalf of the current_user.
- Create a comment on behalf of the current_user.
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/partials/_user.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <div class="user-border user-width">
     <h1><%= link_to user.name, user_path(user) %></h1>
     <div class="user-border-p">
-      <p>Number of posts: <%= user.postsCounter%></p>
+      <p>Number of posts: <%= user.postsCounter || 0%></p>
     </div>
   </div>
 </div>
```
Comment (Emmyn5600) [inline app/views/partials/_user.html.erb:11]: - [ ] There is an issue while loading your project due to the route that is not matching or existing, please fix it.

image
image


- [ ] In the routes, I would recommend keeping the resources.

image

You can keep your routes like this:


`` 
Rails.application.routes.draw do 
  root 'users#index'
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, :new, :create] do
      resources :comments, only: [:new, :create]
      resources :likes, only: [:create]
    end
  end
end 
``

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Ugboaja-Uchechi 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:

- All linters are passing :heavy_check_mark:
- Good commit history :heavy_check_mark:
- Good PR description :heavy_check_mark:

## Required Changes :recycle:

- [ ] Check the comment under the review

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ugboaja-Uchechi** [comment]
To fix this error, Run:
`rails db:create`
`rails db:migrate`
`rails db:seed`
before running `rails s` in your console to view the website

**Review by descholar-ceo — APPROVED** [review]
Hi @Ugboaja-Uchechi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**